use-cases/data-export.md

```diff
@@ -0,0 +1,84 @@
+# Data export
+
+The _Data export_ feature of the **Mews Connector API** provides a streamlined method for retrieving bulk data. This feature is designed for users who require access to a substantial amount of data, such as [Order items](../operations/orderitems.md#order-item), without the hassle of making multiple paginated requests subject to timeouts and other constraints.
+
+### User scenarios
+
+- You are a Mews partner onboarding a new property and you need an initial data sync
+- You are a Mews Multi-Property user and want to retrieve data shared across multiple properties
+- You are using a newly added API operation and are loading data for the first time
+- You wish to perform a cross-data sanity check
+
+### User benefits
+
+- Streamlined bulk data export process
+- Reduced API request complexity and potential timeouts
+- Improved scalability for large-scale data retrieval
+- Faster property onboarding process
+- Potential for future expansion to support further data entities
+
+> ### Restricted!
+>
+> This feature is currently under beta test and is subject to change. If you would like to take advantage of the feature, contact us via [partnersuccess@mews.com](mailto://partnersuccess@mews.com).
+
+## Initiate the export
+
+To initiate a bulk data export, use the [Add export](../operations/exports.md#add-export) operation, specifying the data _Entity type_, e.g. `OrderItem`.
+
+| <div style="width:350px">'How to' use case</div> | API Operations |
+| :-- | :-- |
+| How to create a new bulk data export | [Add export](../operations/exports.md#add-export) |
+
+## Check export status
+
+To monitor the progress of the export and retrieve the exported data, use the [Get all exports](../operations/exports.md#get-all-exports) operation. This operation provides a list of all ongoing and completed exports, including their status.
```

Comment:
suggestion(non-blocking): The operation provides only exports for which the export ID is provided, so I think the use of export ID should be emphasized here.
Additionally I think we could mention periodic polling here? Something along the lines of: "You can repeatedly check the status of export while its status is `Pending` or `Processing`." – Perhaps with some suggestion that it's recommended to check every 5 minutes or so?